scripts/vm/network/security_group.py

```diff
@@ -1029,6 +1046,14 @@ def addFWFramework(brname):
         sys.exit(1)
     cmd = args[0]
     logging.debug("Executing command: " + str(cmd))
+
+    for i in range(0, 30):
```

Comment:
this can be refactored separately as an acquire lock method that may optionally accept a timeout